etc/emscripten/run-web-demo.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -eux
+
+echo This script assumes you have already run 'build.sh'
+
+mkdir -p web-example
+cp etc/emscripten/web-template/* web-example
+cp gap.js gap.wasm gap.data web-example
+cd web-example
+../etc/emscripten/server.rb
```

Comment:
```suggestion
../etc/emscripten/server.rb

```
Add trailing newline